docs: EN quickstart-reference-security (#166)
werf/trdl: docs/pages_en/QUICKSTART.md

Enable the plugin as a `secrets engine` at a specific path in Vault:

```shell
vault secrets enable -path=trdl-test-project vault-plugin-secrets-trdl
```

You can enable the same plugin many times; however, you must use a unique path each time. For more information, refer to the [official documentation](https://www.vaultproject.io/docs/commands/secrets/enable).

Now let's configure the trdl plugin itself. We will use the [/configure](/reference/vault_plugin/configure.html#configure-plugin) API method to do this:

```shell
vault write trdl-test-project/configure @configuration.json
```

where `configuration.json` has the following contents:
```json
{
  "s3_secret_access_key": "FOO",
  "s3_access_key_id": "BAR",
  "s3_bucket_name": "trdl-test-project-tuf",
  "s3_region": "europe-west1",
  "s3_endpoint": "https://storage.googleapis.com",
  "git_repo_url": "https://github.com/werf/trdl-test-project",
  "required_number_of_verified_signatures_on_commit": 2
}
```

> > When configuring the plugin, you must specify the minimum number of GPG signatures required for a commit (`required_number_of_verified_signatures_on_commit`). Otherwise, the release system becomes vulnerable because any user with access can affect operations that turn out to be unprotected by the quorum defined during the configuration.

Reviewer: Второе предложение в оригинале слишком длинное, не сразу въедешь. В переводе его тоже тяжело с первого раза прочитать. Возможно, стоит упростить: «В противном случае система обновлений становится уязвимой: операции контролируются не кворумом, а любым пользователем с доступом».

Author: А как насчет: "_Otherwise, the release system becomes vulnerable: any user with access can tamper with it because the system is not protected by a quorum._" ?

Author: И @olegznv твой вариант: "_Otherwise, the release system becomes vulnerable: operations are not controlled by the quorum, but by any user with access._"

Reviewer: Да, отлично

Author: ```suggestion
> When configuring the plugin, you must specify the minimum number of GPG signatures required for a commit (`required_number_of_verified_signatures_on_commit`). Otherwise, the release system becomes vulnerable: any user with access can tamper with it because the system is not protected by a quorum.
```

The minimum number of GPG signatures required (`required_number_of_verified_signatures_on_commit`) depends on the size and scope of the team, frequency of updates, and other factors.

#### Managing public parts of trusted GPG keys

> The [/configure/trusted_pgp_public_key](/reference/vault_plugin/configure/trusted_pgp_public_key.html) group of API methods is used to handle the public parts of trusted GPG keys..

Reviewer: Лишняя точка

Author: ```suggestion
The [/configure/trusted_pgp_public_key](/reference/vault_plugin/configure/trusted_pgp_public_key.html) group of API methods is used to handle the public parts of trusted GPG keys.
```
